dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -94,6 +95,28 @@ You can always get the latest version of the creative code below from [the AMP e
 
 {% endhighlight %}
 
+### User Sync
+
+To do User Sync on Amp pages, `amp-iframe` element need to be set up on the page as shown below. Below given `amp-iframe` will do User Sync for all the bidders in Prebid Server. We are supporting image pixel syncs as of now, iframe syncs are not supported.
+
+{: .alert.alert-warning :}
+Iframes must be either 600px away from the top or not within the first 75% of the viewport when scrolled to the top – whichever is smaller. For more information on this, see [amp-iframe](https://ampbyexample.com/components/amp-iframe/)
+
+{% highlight html %}
+
+    <amp-iframe width="1" title="User Sync"
+                            height="1"
+                            layout="responsive"
+                            sandbox="allow-scripts"
+                            frameborder="0"
+                            src="https://acdn.adnxs.com/prebid/amp/load-cookie.html?<PLACEMENT_ID1>,<PLACEMENT_ID2>">
```

Comment:
Do we really need to require users to pass all their placement IDs for a given page in the querystring here?